administrator/language/de-DE/de-DE.plg_fields_editor.ini

```diff
@@ -4,5 +4,17 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
 ; Note : All ini files need to be saved as UTF-8
 
+PLG_FIELDS_EDITOR="Felder - Editor"
+PLG_FIELDS_EDITOR_LABEL="Editor (%s)"
+PLG_FIELDS_EDITOR_PARAMS_BUTTONS_HIDE_DESC="Verbirgt die Buttons in der Liste."
```

Comment:
Liste der Editor Buttons welche nicht angezeigt werden sollen.

engl.
Comma separated list of editors-xtd plugin buttons to be hidden